File: pset6/mario/mario.py
from cs50 import get_int

layers = get_int("Height: ")
while layers > 8 or layers < 1:
    layers = get_int("Height: ")
b = layers

for i in range(layers):
    b -= 1
    #
    print(' ' * b, end="")
    print("#" * (i + 1), end="")
    # two space in the middle
    print('  ', end='')
    print("#" * (i + 1), end="")
    print()
    

File: pset1/mariohard/mario.c
#include <stdio.h>
#include <cs50.h>

void stack(int height);

int main(void)
{
    int n ;

    //force user to enter the number within the range
    do
    {
        n = get_int("Height: ");
    }
    while (n > 8 || n < 1);

    stack(n);
}

void stack(int height)
{
    if (height == 0)
    {
        return;
    }

    stack(height - 1);

    for (int i = 0 ;i < height ;i++)
    {
        printf("#");
    }

    printf("  ");

    for (int i = 0 ;i < height ;i++)
    {
        printf("#");
    }

    printf("\n");
}